src/reducers/matches.js

```diff
@@ -20,11 +20,30 @@ import {
 } from '../constants/actionTypes';
 
 const initialState = {
-    
+    fetchingMatches: false,
```

Comment:
this file is going to lead to a merge conflict then; I refactored all the reducers.

Reply:
Can you revert changes in that file? @jdspell